src/views/ApplicationPage/index.js

```diff
@@ -1 +1,193 @@
 // Application page
+// Application page
+import { useState } from "react";
+import { CognitoUserPool } from "amazon-cognito-identity-js";
+import React, { Component } from "react";
+import Form from "react-bootstrap/Form";
+import Button from "react-bootstrap/Button";
+import { Link, Route, Redirect } from "react-router-dom";
+import DatePicker from "react-datepicker";
+import "react-datepicker/dist/react-datepicker.css";
+import Select from "react-select";
+import { SchoolsLVpair } from "./schools.js";
+import {Majors} from './majors.js'
+
+const options = [
```

Comment:
Move these definitions into their own files in the config folder (either replace the ones that are already there or use those), then import them just like you did SchoolsLVpair!